crates/node/primitives/src/client/blob.rs

```diff
@@ -1,10 +1,13 @@
 use std::sync::Arc;
 
 use calimero_blobstore::{Blob, Size};
-use calimero_primitives::blobs::BlobId;
+use calimero_primitives::blobs::{BlobId, BlobInfo, BlobMetadata};
 use calimero_primitives::hash::Hash;
+use calimero_store::key::BlobMeta;
```

Comment:
use `key::BlobMeta` where needed, prefixing `key` helps solve the ambiguity with possible uses of `types::BlobMeta`